api/routes/auth/logout.endpoint.js

```diff
@@ -1,43 +1,22 @@
-const { request, getFullPath } = require('../../utils.endpoint');
+const { api, login } = require('../../endpoint-tests/utils');
 
 describe('logout endpoint | /auth/logout', () => {
-  const url = getFullPath('/auth/logout');
+  const url = '/auth/logout';
 
-  it('not already logged in', async () => {
-    const cookies = request.jar();
+  describe('when unauthenticated', () => {
+    it('returns 400 status', async () => {
+      const response = await api.get(url);
 
-    const { response } = await request.get(url, { jar: cookies });
-
-    expect(response.statusCode).toEqual(200);
-
-    // It might be valid for the API to set some header other than
-    // the session cookie
-    expect(
-      response.headers['set-cookie']
-        .filter(cookie => cookie.startsWith('token='))
-        .every(
-          cookie =>
-            cookie ===
-            'token=; path=/; expires=Thu, 01 Jan 1970 00:00:00 GMT; samesite=none; httponly'
-        )
-    ).toBeTruthy();
+      expect(response.status).toEqual(400);
+    });
```

Comment:
### discussion / maybe change

This seems like a change in behavior. Would it be better for the API to always return a `200` (or probably most appropriately a `204` but 🤷‍♂️) for `/auth/logout`? I'm trying to think of the benefit of signaling to the caller that they weren't already logged in, versus the burden.

E.g., if a user has left the web app open for a while, it's possible that their auth token could expire. Then if they click the logout button in the app, what happens in the app if that results in a non-success status? (At the moment, the app wouldn't do anything because it only proceeds to the logout/re-log-in screen on a success status).